builder/googlecompute/config_test.go

```diff
@@ -685,6 +684,33 @@ func TestConfigExtraBlockDevice_create_image_multiple(t *testing.T) {
 	}
 }
 
+func TestConfigPrepareImageArchitecture(t *testing.T) {
+	cases := []struct {
+		Architecture string
+		ExpectValid  bool
+	}{
+		{"x86_64", true},
+		{"ARM64", true},
```

Comment:
I modified the test case value to be upper case, however did you still want me to have uppercase testing here? I'm unsure if this would be required as only two values are valid and both are uppercase in string.

Reply:
The code is simple enough that we may not need it here, but I tend to err in favour of making sure we don't break it later, so if you're in for a final reroll to address the `else` in the `Prepare` function, I'd suggest adding one case with like `x86_64` or `arm64` to be sure they're not rejected, but if you don't no stress, that's not a blocker for merging afterwards I'd think